README.md

```diff
@@ -15,15 +15,25 @@ npm install testcafe-browser-provider-electron
 
 If your JavaScript application runs in Electron, follow these steps to set up testing.
 
-1. Create a `.testcafe-electron-rc` file in the root application directory. Include the following settings to this file.
+1. Create a JSON `.testcafe-electron-rc` or JavaScript `.testcafe-electron-rc.js` [configuration file](#configuration) in the root application directory. Include the following settings to this file.
```

Comment:
We can also use the `.json` extension for JSON config files. Actually, I think we should always use an explicit extension, so we can add the extension everywhere in this file and do not mention extensionless config files as it is a legacy feature.

Reply:
```suggestion
1. Create a JSON `.testcafe-electron-rc.json` or JavaScript `.testcafe-electron-rc.js` [configuration file](#configuration) in the root application directory. Include the following settings to this file.
```